packages/smooth_app/lib/data_models/abstract_onboarding_data.dart

```
@@ -30,14 +30,20 @@ abstract class AbstractOnboardingData<T> {
   ///
   /// Typical use case: data we downloaded just for the onboarding,
   /// that we can clear after the onboarding.
+  /// Returns true if error thrown while loading
```

Probably not there. And "Returns true if error" is hard to swallow.